frontend/src/screens/learner.tsx

```diff
@@ -145,81 +164,95 @@ export const MobileStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies })
                     perSlideOffset: 14,
                 }}
                 centeredSlides={true}
-                pagination={{
-                    enabled: true,
-                    dynamicBullets: true,
-                    dynamicMainBullets: 5,
-                }}
-                modules={[EffectCards, Pagination]}
+                modules={[EffectCards, Navigation]}
                 style={{
                     paddingBottom: '2rem',
                     marginBottom: '1rem',
                 }}
+                pagination={false}
+                navigation={true}
             >
-                {studies.map((study) => (
+                {studies.sort((a, b) => (a.completedAt === b.completedAt) ? 0 : a.completedAt ? 1 : -1).map((study) => (
                     <SwiperSlide key={study.id} className="pb-1">
                         <StudyCard study={study} />
                     </SwiperSlide>
                 ))}
             </Swiper>
+            <style>
```

Comment:
we probably don't want inline CSS here - could we extract this out to a .css file? we may already have one - worth checking